client/job.lua

```diff
@@ -401,13 +401,13 @@ CreateThread(function()
                 end
             end
 
-            local currentHospital = 1
+           local currentHospital = 1
 
             for k, v in pairs(Config.Locations["main"]) do
                 local dist = #(pos - v)
                 if dist < 1.5 then
-                    sleep = 5
-                    DrawText3D(v.x, v.y, v.z, Lang:t('text.elevator_roof'))
+                    sleep = 7
+                    DrawText3D(v.x, v.y, v.z, "~g~E~w~ - Take the elevator to the roof")
```

Comment:
Wrong, you speak english but some people using this script don't.

Reply:
since when is ML a thing?